services/job_skill_service.py

```
@@ -0,0 +1,266 @@
+
+                # Create job skill relationship if it doesn't exist
+                job_skill_result = self.create_job_skill(job_id, skill.id)
+                if is_success(job_skill_result):
+                    created_job_skills.append(job_skill_result.obj)
+                elif "already exists" not in job_skill_result.error:
+                    current_app.logger.warning(f"Failed to create job skill relationship: {job_skill_result.error}")
+
+            current_app.logger.info(f"Extracted and created {len(created_job_skills)} job skills for job {job_id}")
+            return success_result(created_job_skills)
+
+        except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)